rosetta/infra/harmony-pstn.conf

```diff
@@ -120,3 +120,10 @@ Version = "2.5.11"
   Enabled = true
   IP = "0.0.0.0"
   Port = 9800
+
+[GasPrice]
+  Blocks = 20
+	Percentile = 80
+	DefaultPriceGwei = 9
+	MaxPriceGwei = 12
+  NumberTxsSampled = 3
```

Comment:
Indent

Reply:
Fixed. Not sure why tabs weren't working so I switched it to double space.